src/system/ThemeModule/Resources/views/Config/config.html.twig

```diff
@@ -18,4 +18,66 @@
     </div>
 </div>
 {{ form_end(form) }}
+{% macro imageRow(name, label, description, code, isImage = true) %}
+    {% set useImageFunction = name in ['logo', 'mobileLogo', 'icon'] %}
+    {% set path = useImageFunction ? siteImagePath(code) : asset(code) %}
+    <tr>
+        <th id="h{{ name|capitalize }}" scope="row" headers="hType">{{ label }}</th>
+        <td headers="h{{ name|capitalize }} hDescription">{{ description }}</td>
+        <td headers="h{{ name|capitalize }} hPath">{{ path }}</td>
```

Comment:
maybe do not allow text wrapping in the path column. I was confused at first when I looked at the graphic

Reply:
then the table needs horizontal scrolling which I wanted to avoid